apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -194,7 +196,11 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
                         <tr>
                             <td className={classes.alignToTop}>Location</td>
                             <td className={`${classes.multiline} ${classes.rightCells}`}>
-                                <Link className={classes.clickableLocation} to={`/map?location=${buildingId}`}>
+                                <Link
+                                    className={classes.clickableLocation}
+                                    to={`/map?location=${buildingId}`}
+                                    onClick={focusMap}
```

Comment:
Monkeying it a bit yeah?

Reply:
Context API moment :sunglasses:
It's honestly not the worst strat, but it is most definitely a monkey patch!

Reply:
Better than the way I did it.